_dev/apps/ui/stories/product-feed-page-components-configuration.stories.ts

```diff
@@ -0,0 +1,27 @@
+import FeedConfigurationCard from '@/components/product-feed-page/feed-configuration/feed-configuration-card.vue';
+
+export default {
+  title: 'Product-Feed-Page/Product Feed Page/Components/Feed Configuration',
+};
+
+const ProductFeed = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { FeedConfigurationCard },
+  template: `<FeedConfigurationCard
+    ref="FeedConfigurationCard"
+    v-bind="$props"
+  />`,
+});
+
+export const Default:any = ProductFeed.bind({});
+Default.args = {
+  productFeedConfiguration: {
+    lastModificationDate: new Date(),
```

Comment:
Need to specify a specific date to avoid detected changes on the interface at each commit